pyproject.toml

```diff
@@ -41,15 +41,15 @@ classifiers = [
 "Bug Tracker" = "https://github.com/chaoss/grimoirelab-perceval-puppet/issues"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 
 requests = "^2.7.0"
 grimoirelab-toolkit = { version = ">=0.3", allow-prereleases = true}
 perceval = { version = ">=0.19", allow-prereleases = true }
 
 [tool.poetry.dev-dependencies]
-httpretty = "1.0.2"
-flake8 = "^4.0.1"
+httpretty = "^1.0.2"
```

Comment:
We have a different version of httpretty in the other backends.

Reply:
Fixed, it will use the same version because it is `^1.1.4` in other backends, but it's better to be the same.